www/index.html

```diff
@@ -54,44 +54,32 @@
         <section id="search-article" role="region">
             <header id="top">
                 <nav class="navbar navbar-expand-md bg-light" role="navigation">
-                    <a class="navbar-brand">Kiwix 2.7-WIP</a>
-
+                    <!--<a class="navbar-brand">Kiwix 2.7-WIP</a> -->
```

Comment:
Replacing the text by the icon is a good move.
On the other hand, it means that we're now missing the version information. Maybe it could simply be added in the About/Information section?

By the way, the whole line can be removed instead of commented